search-provider/eks-discovery-feed-provider.c

```diff
@@ -195,12 +195,20 @@ add_key_value_int_to_str_pair_from_model_to_variant (EkncContentObjectModel *mod
 }
 
 static void
-add_author_from_model_to_variant (EkncArticleObjectModel *model,
-                                  GVariantBuilder        *builder,
-                                  const char             *key)
+add_first_string_value_from_model_to_variant (EkncContentObjectModel *model,
+                                              GVariantBuilder        *builder,
+                                              const char             *model_key,
+                                              const char             *key)
 {
-  char * const *authors = eknc_article_object_model_get_authors (model);
-  add_key_value_pair_to_variant (builder, key, authors ? authors[0] : "");
+  g_auto(GStrv) values;
```

Comment:
This will free data belonging to the model.

Reply:
Oh, does it only give a shallow reference through `g_object_get`? I'll fix that so that we don't free model memory.

Reply:
Oh, this may have been my bad.

Reply:
Indeed, I missed that you switched from the accessor function to `g_object_get`. Never mind.

Reply:
(reverted the change to make the pointer non-owned)